### PR TITLE
Set focus to top after filtering

### PR DESCRIPTION
**Pull request by ajpal**
@dsjen @thallada

### CODE REVIEW

File: analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js [hunk 1]
```diff
@@ -741,6 +729,14 @@ define(function(require) {
                     });
                 });
 
+                it('sets focus to the top after filtering', function() {
+                    var filterFieldName = this.filterFieldName;
+                    getRosterView({courseMetadataModel: this.courseMetadata});
+                    spyOn($.fn, 'focus');
+                    expectCanFilterBy(filterFieldName, this.firstFilterOption);
+                    expect($('#learner-app-focusable').focus).toHaveBeenCalled();
```
Comment (thallada) [inline analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js:737]: Can we actually check if the element is focused here (and the test below)? e.g.:

```
expect($('#learner-app-focusable')).toBeFocused();
```

**Comment by thallada** [comment]
I think we should figure out how to trigger methods in grandparent views at some point, but setting the focus is simple enough that I think it's fine to duplicate.

I also wish there was a good way to save the element `$('#learner-app-focusable')` as a constant variable somewhere and reference it in all the views, but I don't know how to do that.

Anyways, lgtm :+1:

File: analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js (hunk 1 above)
Comment (ajpal) [inline analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js:737]: Agreed, but it looks like that doesn't work in this case. I followed the example [here](https://github.com/edx/edx-analytics-dashboard/pull/518/files#diff-180efbbbb1163c8babf958c1f2158fdbL1052)

Comment (dsjen) [inline analytics_dashboard/static/apps/learners/roster/views/spec/roster-spec.js:737]: :(  I have vague memories of `.toBeFocused()` not working too and it seems like it's known to be a bit flaky: http://stackoverflow.com/questions/20564143/how-to-test-whether-an-input-form-field-is-focussed.

**Comment by dsjen** [comment]
@thallada -- yes, I'm totally with you on needing to resolve the trigger, but tabling that for now.

@ajpal -- thanks for the simplicity!  👍

**Pull request merged**